package.json

```diff
@@ -450,27 +449,15 @@
     "webpack-dev-server": "^4.11.1",
     "webpack-node-externals": "^3.0.0",
     "xterm": "^5.0.0",
-    "xterm-addon-fit": "^0.5.0"
+    "xterm-addon-fit": "^0.5.0",
```

Comment:
Why is this commit part of this monster PR at all? Doesn't have anything to do with fixing something, it's adding stuff.